src/screen/dashboard/sidebar.rs

```diff
@@ -108,6 +108,129 @@ impl Sidebar {
             }
         }
     }
+
+    fn menu_buttons<'a>(
+        &self,
+        main_window: window::Id,
+        panes: &Panes,
+        config: data::config::Sidebar,
+        show_tooltips: bool,
+        file_transfers: &'a file_transfer::Manager,
+        version: &'a Version,
+        theme_editor_open: bool,
+    ) -> Element<'a, Message> {
+        let mut menu_buttons = row![]
+            .spacing(1)
+            .width(Length::Shrink)
+            .align_y(Alignment::End);
+
+        let tooltip_position = match config.position {
+            sidebar::Position::Top => tooltip::Position::Bottom,
+            sidebar::Position::Bottom | sidebar::Position::Left | sidebar::Position::Right => {
+                tooltip::Position::Top
+            }
+        };
+
+        // the menu_buttons = menu_buttons.push(...) cannot be refactored into this closure, due to
+        // move semantics
```

Comment:
I think we can remove this comment, as it doesn't provide that much information.